src/app/pages/task-calendar/cron/cron-form/cron-form.component.ts

```diff
@@ -84,16 +85,16 @@ export class CronFormComponent {
            type: 'checkbox',
            name: 'cron_stdout',
            placeholder: T('Redirect Standard Output'),
-           tooltip: T('Set to disable emailing standard output (stdout) to the\
-             <i>root</i> user account.'),
+	   tooltip: T('Set to discard stdout (standard output) instead\
```

Comment:
This changed in meaning, from not being mailed to the *root* user, to not being mailed to the selected user.  Is that correct?  The description is confusing, too, with two different meanings for the redundant use of "set".

Maybe:
Redirect stdout to null.  When unset, output from the command is mailed to the user (or root? verify) running the cron job.

Reply:
Spoke to William and confirmed elsewhere that the user that owns/runs the cron is who gets stderr/stdout. Changed from root to user to reflect this.